charts/csm-authorization/templates/proxy-server.yaml

```diff
@@ -114,4 +114,4 @@ spec:
   - name: http
     protocol: TCP
     port: 8080
-    targetPort: 8080
+    targetPort: 8080
```

Comment:
Newline issue.

Reply:
Not really, YAML doesn't need an ending line feed, but I added it back in

Reply:
Thanks!